packages/hardhat/contracts/Ethernauts.sol

```diff
@@ -78,7 +86,7 @@ contract Ethernauts is ERC721Enumerable, Ownable {
     // --------------------
 
     function getCurrentSaleState() public view returns (SaleState) {
```

Comment:
Is this function useful? Since currentSaleState is now public, I don't think so.

Reply:
updated and replaced with the autogenerated function in tests